### PR TITLE
Add ad-hoc returns enter licence page

### PR DESCRIPTION
**Pull request by Beckyrose200**
https://eaflood.atlassian.net/browse/WATER-4816

As part of the work for sending ad-hoc returns, this PR focuses on adding the enter a licence page to the journey.

### CODE REVIEW

File: app/services/notifications/ad-hoc-returns/submit-licence.service.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+  const dueReturns = await ReturnLogModel.query().where('licenceRef', licenceRef).where('status', 'due').first()
+
+  return !!dueReturns
+}
+
+async function _fetchLicence(licenceRef) {
+  const licence = await LicenceModel.query().where('licenceRef', licenceRef).first()
+
+  return !!licence
+}
```
Comment (StuAA78) [inline app/services/notifications/ad-hoc-returns/submit-licence.service.js:62]: Just a small thing but I'd be expecting a function called "fetch..." to fetch _and return_ something. These are more like `_dueReturnsExist` and `_licenceExists` or something like that.

**Review by StuAA78 — CHANGES_REQUESTED** [review]
Looks good, just a couple of tiny tweaks for clarity requested 🙂

File: app/services/notifications/ad-hoc-returns/initiate-session.service.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+'use strict'
```
Comment (Jozzey) [inline app/services/notifications/ad-hoc-returns/initiate-session.service.js:1]: Was there a reason why we couldn't use `app/services/bill-runs/setup/initiate-session.service.js` to do this if we moved it to a more generic folder?

**Review by StuAA78 — APPROVED** [review]

**Pull request merged**